Ginger/GingerCoreCommon/Actions/ActLogAction.cs

```diff
@@ -60,10 +60,20 @@ public string LogText
             }
         }
 
+        public enum LogLevel
+        {
+            [EnumValueDescription("Information")]
+            INFO,
+            [EnumValueDescription("Warning")]
+            WARN,
+            [EnumValueDescription("Error")]
+            ERROR
+        }
+
         [IsSerializedForLocalRepository]
-        public eLogLevel SelectedLogLevel { get; set; }
+        public LogLevel SelectedLogLevel { get; set; }
 
-        public eLogLevel LogTypes { get; set; }
+        public LogLevel LogTypes { get; set; }
```

Comment:
why "LogTypes" is needed? for binding? why SelectedLogLevel can't be used for it?